Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM atddocker/atd-oracle-py:production
+
+# Copy our own application
+WORKDIR /app
+COPY . /app
+
+RUN chmod -R 755 /app/*
+
+# # Proceed to install the requirements...do
+RUN cd /app && apt-get --allow-releaseinfo-change update 
```

Comment:
I've copied this from some other repo we were using cx_Oracle, so you want me to change this to:?

```
RUN apt-get --allow-releaseinfo-change update 
```

Reply:
Right on - I totally dig that this came from other sources. My suggestion was really intended for clarification, because I don't think we want to imply a link between changing the present working directory with updating the image with apt. 

I would do something like:

```
RUN cd /app && apt-get --allow-releaseinfo-change update
RUN pip install -r requirements.txt
```
↓
```
RUN apt-get --allow-releaseinfo-change update
RUN pip install -r requirements.txt
```

On line 4 of the `Dockerfile` the WORKDIR directive is already used to set the correct PWD. I tried those changes above, and it successfully built on my machine, FWIW.